assets/js/components/setup/SetupUsingProxyWithSignIn.js

```diff
@@ -302,6 +308,22 @@ export default function SetupUsingProxyWithSignIn() {
 
 												<p className="googlesitekit-setup__description">
 													{ description }
+													{ learnMore && (
+														<Fragment>
+															{ ' ' }
+															<Link
+																href={
+																	learnMore
+																}
+																external
+															>
+																{ __(
+																	'Learn More',
```

Comment:
There's already a `"Learn more"` string throughout the codebase, so let's use that instead so this doesn't show up as a new string. 

```suggestion
																	'Learn more',
```

Reply:
Updated, thank you @tofumatt!